ksp-processors/src/main/kotlin/kordenum/generator/Classes.kt

```diff
@@ -0,0 +1,7 @@
+package dev.kord.ksp.kordenum.generator
+
+import com.squareup.kotlinpoet.ClassName
+import com.squareup.kotlinpoet.MemberName
+import com.squareup.kotlinpoet.asClassName
+import kotlinx.serialization.KSerializer
+
```

Comment:
what's this file?

Reply:
Was supposed to contain ClassNames, however I forgot to remove it